Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -79,7 +80,7 @@
                                                     infoClass="text_blue" showDetail="true" showSummary="true"
                                                     tooltip="true"/>
 
-                                        <!-- Box für die Bearbeitung der Details -->
+                                        <!-- Box f�r die Bearbeitung der Details -->
```

Comment:
The simplest solution for this and all other comments is using English without umlauts (as required by the coding conventions).